util/monitoring.js

```diff
@@ -0,0 +1,111 @@
+const _ = require('lodash');
+const googleMonitoring = require('@google-cloud/monitoring');
+const fs = require('fs');
+
+function getValueFromPoint(point) {
+  const valueKey = _.get(point, 'value.value');
+  return _.get(point, `value.${valueKey}`);
+}
+
+async function requestAndAssembleInfo(client, request) {
+  const [timeSeriesList] = await client.listTimeSeries(request);
+  if (timeSeriesList.length === 0) {
+    return {
+      err: `Can't find information`,
+    };
+  }
+  const timeSeries = timeSeriesList[0];
+  const points = _.get(timeSeries, 'points');
+  const info = {};
+  info.start = getValueFromPoint(points[0]);
```

Comment:
Should the start be the startTime of the request? What'd be the main use case of this value?

Reply:
info.start means the value at the start of the test. For example 'monitoring.cpuUsage.start' means cpu usage of startTime. We can see the difference between the performance values of startTime and EndTime.

Reply:
I see. Thanks for the explanation.